logging/src/main/java/alluxio/logging/shell/FsChgrpFlowLoggingAspect.java

```diff
@@ -0,0 +1,41 @@
+package alluxio.logging.shell;
+
+import alluxio.logging.shell.base.FsFlowLoggingAspect;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+
+/**
+ * This class represents a {@link alluxio.logging.base.FlowLoggingAspect} for the
+ * file system command {@code chgrp}. All methods triggered by this command are logged
+ * and kept track of.
+ */
+@Aspect
+public final class FsChgrpFlowLoggingAspect extends FsFlowLoggingAspect {
+
+    private static final String FLOW_NAME = "FsChgrpFlow";
+    private static final String START_METHOD = "execution(* alluxio.cli.fs.command.ChgrpCommand.run(..))";
+
+    @Around(START_METHOD)
+    @Override
+    public Object startFlux(final ProceedingJoinPoint point) throws Throwable {
+        return super.startFlux(point);
+    }
+
+    @Around(WHITE_AND_BLACK_LIST)
+    @Override
+    public Object around(final ProceedingJoinPoint point) throws Throwable {
+        return super.around(point);
+    }
+
+    @Around(FINISH_METHOD)
+    @Override
+    public Object finishFlux(final ProceedingJoinPoint point) throws Throwable {
```

Comment:
Please, let's never use the word "flux", always use "flow".
Please, do it for all methods you created with this name.